README.md

```diff
@@ -24,7 +24,10 @@ Here are some of the benefits of using this library:
 Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
 
 ### Feature Filters
-Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled. At this point the feature is considered enabled and traversal through the feature filters stops. If no feature filter indicates that the feature should be enabled, then it will be considered disabled.
+Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled or disabled. Traversal works in one of two ways:
+
+1. For features with a `RequirementType` of `Any` (the default): The feature-filters are traversed until one of the filters decides that the feature should be enabled. If no filter indicates that the feature should be enabled, then it will be considered disabled.
+2. For features with a `RequirementType` of `All`: The feature-filters are traversed until one of the filters decides that the feature should be disabled. If no filter indicates that the feature should be disabled, then it will be considered enabled.

```

Comment:
Haha, I previously had noted that, but Avani called out that we already address that elsewhere. https://github.com/microsoft/FeatureManagement-Dotnet/pull/226#discussion_r1166119204

Reply:
Where is that other place it is addressed?

Reply:
https://github.com/microsoft/FeatureManagement-Dotnet#feature-filters

> If no feature filter indicates that the feature should be enabled, then it will be considered disabled.

And https://github.com/microsoft/FeatureManagement-Dotnet#feature-flag-declaration

> FeatureU has no filters in its EnabledFor property and thus will never be enabled.

Reply:
I still worry. It's not quite clear that the same is true for the all requirement. If one were in doubt, and they read this description of how all works, then I have a strong feeling that they would be led astray.

Reply:
I'm okay with adding it here explicitly.